fab/fab/fabfile.py

```diff
@@ -222,16 +223,19 @@ def development():
     """
     Must pass in the 'inventory' env variable,
     which is the path to an ansible inventory file
-    and an 'environment' env variable,
+    and an 'deploy_env' env variable,
     which is the name of the directory to be used under /home/cchq/www/
 
     Example command:
 
         fab development awesome_deploy \
-        --set inventory=/path/to/commcare-cloud/ansible/inventories/development,environment=dev
+        --set inventory=/path/to/commcare-cloud/ansible/inventories/development,deploy_env=dev
 
     """
-    load_env('development')
+    # not sure if this still works, it was broken for a while without anyone saying anything
+    # so guessing we can delete this task altogether? ~ Danny
```

Comment:
probably doesn't get used very often (or ever)

Reply:
awesome, good to know.